Base/QTCore/qSlicerExtensionsManagerModel.cxx

```diff
@@ -1741,6 +1741,12 @@ bool qSlicerExtensionsManagerModel::installExtension(
   const ExtensionMetadataType& extensionIndexMetadata =
     Self::parseExtensionDescriptionFile(extensionIndexDescriptionFile);
 
+  // Enable or disable the extension if state was not already set
+  if (!extensionMetadata.contains("enabled"))
+    {
+    extensionMetadata.insert("enabled", d->NewExtensionEnabledByDefault);
+    }
+
```

Comment:
This might be here due to resolving some merge conflicts incorrectly. This diff would be a partial revert of https://github.com/Slicer/Slicer/commit/fb23d84810549856ebc73f7a9f1c95793bf48fc1. You should just remove this diff change from your 1 commit now `ENH: Improve control point state management and add markups toolbar`.

Reply:
Got it, thank you!